docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -666,6 +680,12 @@ The acceptable flags include all those of `kubectl` except for
 - `-X`: turn on debug echoing of commands inside the script that
   implements this command.
 
+This command is sensitive to the followig environment variables.
```

Comment:
"sensitive" is not clear here:  maybe "this command relays on the SM_CONFIG and SM_CONTEXT variables. Make sure they are set before running the command" ? Or something like that ..

Reply:
fixed